src/components/settings/General.vue

```diff
@@ -37,6 +37,42 @@
                 </select>
             </div>
 
+            <!-- Time Format -->
+            <div class="mb-4">
+                <label class="form-label">
+                    {{ $t("Time Format") }}
```

Comment:
Please add all translatable strings to `en.json`, otherwise our translators will not be able to pick this up